data_pipeline/data_processing/aggregatedcsv2redis.py

```diff
@@ -0,0 +1,87 @@
+"""
+Do not merge. For discussion only.
+Alternative way is to use awk and redis-cli
+"""
+import logging
+import os
+import sys
+
+import pandas as pd
+from rejson import Client, Path
+
+logger = logging.getLogger(__name__)
+
+
+class Csv2Redis:
+
+    def __init__(self):
+        self.rj = Client(host=os.environ.get("REDIS_HOST", "localhost"), decode_responses=True)
+
+    def setElectionShapeInRedis(self, filename: str) -> list:
+        '''
+        filename: str Input aggregated csv file. Csv seems to be read faster then Excel.
+        Populate election shape into redis
+        '''
+        with self.rj.pipeline() as pipe:
+            # TODO: assure filename exists
+            # TODO: more error checking
+            # TODO: resolve incremental update
+            try:
+                # TODO: the row pass 1440 generates an error
+                # Get the csv file
+                data = pd.read_csv(filename, skiprows=lambda x: x % 2 == 1, sep=',', quotechar='"', )
+                # There are 4 types RCPT, EXPN, LOAN, S497.
+                # TODO: Get a better understanding of how Loan, Exp & S497 work. We could have simply use Rec_Type for raise or expens but leave it as is for now until we have better understanding of Rec_Type.
+                data['Raise_or_Expense'] = data.apply(lambda row: 'raise' if row['Rec_Type'] == 'RCPT' else (
+                    'expense' if row['Rec_Type'] == 'EXPN' else 'loanors497'), axis=1)
+                # Clean up the Amount that contains $ and ,
+                data['Amount'] = data['Amount'].str.replace(',', '').replace('$', '').astype(float)
```

Comment:
The csv file outputted by scraper also has apostrophes in the amount column: '
Can this also replace apostrophe with ''? Something like replace('\'', '')